code/game/objects/effects/glowshroom.dm

```diff
@@ -39,9 +39,10 @@
 	else //if on the floor, glowshroom on-floor sprite
 		icon_state = "glowshroomf"
 
-	spawn(delay)
-		set_light(round(potency/10))
-		// Spread() - Methinks this is broken - N3X
+	set_light(round(potency/10))
```

Comment:
Fug I shoulda spawn()'d this off now that I look at the affected code a bit more.